### PR TITLE
FIX/adjust env files

### PR DESCRIPTION
**Pull request by JaneMcBrain**
- Removes redundant variables
- removes wrong semantics-url

### CODE REVIEW

**Review by drcgjung — APPROVED** [review]
LGTM @LuLeRoemer  please check/merge with split services approache

File: portal/code/tractus-x-portal/dev042.env [hunk 1]
```diff
@@ -7,7 +7,6 @@ REACT_APP_KEYCLOAK_URL=https://catenaxdev042akssrv.germanywestcentral.cloudapp.a
 REACT_APP_BASIC_SERVICES_AUTHENTICATION=VHJhY3R1c1g6NFpxQTo9N003Z3lySFR6Tg==
 REACT_APP_BUSINESSPARTNER_SERVICE_URL=https://catenaxdev042akssrv.germanywestcentral.cloudapp.azure.com/businesspartners/businesspartner
 REACT_APP_PORTAL_URL=https://catenaxdev042aksportal.germanywestcentral.cloudapp.azure.com/
-REACT_APP_SEMANTIC_SERVICE_LAYER_URL=https://catenaxdev042akssrv.germanywestcentral.cloudapp.azure.com/semantics/api/v1/
```
Comment (LuLeRoemer) [inline portal/code/tractus-x-portal/dev042.env:10]: What is this configuration used for?

**Review by LuLeRoemer — APPROVED** [review]
LGTM
As far as I can tell, the .env configuration that is important for the deployment on INT and dev042 is produced on the fly in the `portal.yml`workflow and the changes related to the split are implemented on code level.
Why did you delete the `REACT_APP_SEMANTIC_SERVICE_LAYER_URL` variable from the `dev042` and `int` files? In general I am not sure what these files are used for.

File: portal/code/tractus-x-portal/dev042.env (hunk 1 above)
Comment (JaneMcBrain) [reply portal/code/tractus-x-portal/dev042.env:10]: > LGTM As far as I can tell, the .env configuration that is important for the deployment on INT and dev042 is produced on the fly in the `portal.yml`workflow and the changes related to the split are implemented on code level. Why did you delete the `REACT_APP_SEMANTIC_SERVICE_LAYER_URL` variable from the `dev042` and `int` files? In general I am not sure what these files are used for.

As the description says: these variales are redundant. You will find these variables still in the file.

**Pull request merged**